pages/contribute/style_guide.md

```diff
@@ -51,8 +51,8 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
 
 ## Bibliography
  * Add your citations as bibtex to the `_bibliography/references.bib` file.
- * Add `{% cite reference_key %}` to the text where you are citing the reference.
- * Add ## Bibliography {% bibliography --cited %} to show a bibliography section containing the references in the page.
+ * Add {% raw %} {% cite reference_key %} {% endraw %} to the text where you are citing the reference.
+ * Add {% raw %} ## Bibliography {% bibliography &#8209;&#8209;cited %} {% endraw %} to show a bibliography section containing the references in the page.
```

Comment:
```suggestion
 * Add {% raw %} `{% cite reference_key %}` {% endraw %} to the text where you are citing the reference.
 * Add {% raw %} `## Bibliography {% bibliography --cited %}` {% endraw %} to show a bibliography section containing the references in the page.
```

Do you have an opinion on adding it to quotes? don't know if it add something for the readability.


![image](https://github.com/elixir-europe/rdmkit/assets/44875756/b00dbd7d-96f4-4f87-932c-184e4d612494)